### DISCUSSION TITLE
EJBCA CE 8.3

### DISCUSSION THREAD

**Discussion by hesunmark** [Keyfactor/ejbca-ce#609] [Announcements]
EJBCA Community 8.3 is released

We are pleased to announce the release of EJBCA Community 8.3.

Release highlights include: 

- Hybrid Certificates
- Bouncy Castle upgraded to 1.78
- EJBCA updated with top-level menu
- Documentation on Keyfactor Docs

For more information: 

Read the full [EJBCA Community 8.3 Release Notes](https://docs.keyfactor.com/ejbca/latest/ejbca-community-8-3-release-notes)

Visit the EJBCA project on [ejbca.org](https://www.ejbca.org/)

<hr /><em>This discussion was created from the release <a href='https://github.com/Keyfactor/ejbca-ce/releases/tag/r8.3.1'>EJBCA CE 8.3</a>.</em>

**Comment by RufusJWB** [comment]
Wonderful! Do we have a chance of getting a container image at https://hub.docker.com/r/keyfactor/ejbca-ce/tags ?

**Reply by hesunmark** [reply]
For sure! On its way. We need to do a couple of adjustments but I'd expect it this week.
